### PR TITLE
AppLab widget mode styling clean-up

### PR DESCRIPTION
**Pull request by maddiedierker**
Follow-up to #40298.

More pre-factoring for [STAR-1549](https://codedotorg.atlassian.net/browse/STAR-1549) (allowing levelbuilders to create AppLab widgets with start & design modes). A few things happening here:

1. Consolidate AppLab app width calculation into a function (`getAppWidth` in applab/constants.js).
2. Consolidate widget mode styles to remove unnecessary conditionals where possible.
3. Remove `!important` from `.widgetWidth` and `.widgetHeight` classes using specificity rules.
4. General organization and clean-up.

## Links

- [STAR-1549](https://codedotorg.atlassian.net/browse/STAR-1549)

## Testing story

Relies on existing storybook and eyes tests.

### CODE REVIEW

File: apps/src/applab/applab.js [hunk 1]
```diff
@@ -149,15 +149,7 @@ function loadLevel() {
   Applab.timeoutFailureTick = level.timeoutFailureTick || Infinity;
   Applab.minWorkspaceHeight = level.minWorkspaceHeight;
   Applab.softButtons_ = level.softButtons || {};
-
-  // Historically, appWidth and appHeight were customizable on a per level basis.
-  // This led to lots of hackery in the code to properly scale the visualization
-  // area. Width/height are now constant, but much of the hackery still remains
-  // since I don't understand it well enough.
```
Comment (maddiedierker) [inline apps/src/applab/applab.js:156]: This comment is outdated / no longer needed. It was originally added in #7721, but the hackery was addressed in #14312.

File: apps/style/applab/style.scss [hunk 2]
```diff
@@ -231,16 +233,6 @@ div#visualizationResizeBar {
   }
 }
 
-.widgetWidth {
```
Comment (epeach) [inline apps/style/applab/style.scss:234]: :tada Yay for removing !important!

File: apps/src/applab/Visualization.jsx [hunk 3]
```diff
@@ -58,35 +59,35 @@ class Visualization extends React.Component {
...
-  };
-
   handleDisableMaker = () => project.setMakerEnabled(null);
 
   handleTryAgain = () => {
     studioApp().resetButtonClick();
     studioApp().runButtonClick();
   };
 
+  getVisualizationClassNames = () => {
```
Comment (epeach) [inline apps/src/applab/Visualization.jsx:69]: Breaking this out is nice.

**Review by epeach — APPROVED** [review]
Looks great after pulling out all those repeated ternaries into a function!

**Pull request merged**